### ISSUE TITLE
[Deployment Issue] CapRover Unresponsive After `serversetup`

### ISSUE THREAD

**Issue by AaronPresley** [caprover/caprover#1404]
Hi, thank you for this great app! I've done my best to figure out the problem myself be following the [Troubleshooting guides](https://caprover.com/docs/troubleshooting.html) but I'm unfortunately still stuck.

**What is the problem?**

I'm able to spin up a new instance (Linode, `g6-nanode-1`), install Docker, and see the UI at `<my ip>:3000`. I'm also able to successfully run `caprover serversetup` which gives me positive output as if things worked as expected.

However, from this point on the new url continues to load until I get a `ERR_CONNECTION_TIMED_OUT` error in the browser. Similar result if I go to `<my up>:3000`.

I would greatly appreciate some guidance 🙏  Below you will see the output of various commands as written out in the Troubleshooting guide. Note that I haven't modified or made any custom nginx configs

```
$ ufw allow 80,443,3000,996,7946,4789,2377/tcp; ufw allow 7946,4789,2377/udp;
Skipping adding existing rule
Skipping adding existing rule (v6)
Skipping adding existing rule
Skipping adding existing rule (v6)
```

```
$ docker ps
CONTAINER ID   IMAGE                              COMMAND                  CREATED          STATUS          PORTS                                                                      NAMES
af0fafa286a1   caprover/certbot-sleeping:v1.6.0   "/bin/sh -c 'sleep 9…"   10 minutes ago   Up 9 minutes    80/tcp, 443/tcp                                                            captain-certbot.1.0nfqobt4a2v7zwou6itjkn2p4
91fa814fef11   nginx:1                            "/docker-entrypoint.…"   10 minutes ago   Up 10 minutes   0.0.0.0:80->80/tcp, :::80->80/tcp, 0.0.0.0:443->443/tcp, :::443->443/tcp   captain-nginx.1.zf9sqk7q7xisi2lq5jutpylmi
b51aaebdb3b7   caprover/caprover:1.10.1           "docker-entrypoint.s…"   10 minutes ago   Up 10 minutes   0.0.0.0:3000->3000/tcp, :::3000->3000/tcp                                  captain-captain.1.nemlno8mk4lgkwebcaifz7ict
```

```
$ docker service ps captain-captain --no-trunc
ID                          NAME                    IMAGE                      NODE        DESIRED STATE   CURRENT STATE             ERROR                                                              PORTS
nemlno8mk4lgkwebcaifz7ict   captain-captain.1       caprover/caprover:1.10.1   localhost   Running         Running 10 minutes ago                                                                       *:3000->3000/tcp,*:3000->3000/tcp
monk6ufcysa7rvddgk0x58ods    \_ captain-captain.1   caprover/caprover:1.10.1   localhost   Shutdown        Rejected 11 minutes ago   "cannot create a swarm scoped network when swarm is not active"
qc7smslu8n9il7bggddeh4cdd    \_ captain-captain.1   caprover/caprover:1.10.1   localhost   Shutdown        Failed 11 minutes ago     "No such container: captain-captain.1.qc7smslu8n9il7bggddeh4cdd"
ctz4u92o667xxn8xlk4pe68sp    \_ captain-captain.1   caprover/caprover:1.10.1   localhost   Shutdown        Shutdown 11 minutes ago
h0w7tfmnmespj51g3u0rdstcj    \_ captain-captain.1   caprover/caprover:1.10.1   localhost   Shutdown        Shutdown 11 minutes ago
```

```
curl localhost:3000 -v
* Rebuilt URL to: localhost:3000/
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 3000 (#0)
> GET / HTTP/1.1
> Host: localhost:3000
> User-Agent: curl/7.58.0
> Accept: */*
>
< HTTP/1.1 302 Found
< X-Powered-By: Express
< Location: https://localhost:3000/
< Vary: Accept
< Content-Type: text/plain; charset=utf-8
< Content-Length: 45
< Date: Fri, 29 Apr 2022 22:34:17 GMT
< Connection: keep-alive
< Keep-Alive: timeout=5
<
* Connection #0 to host localhost left intact
Found. Redirecting to https://localhost:3000/
```

Output of the following command on your server:

```
uname -a && lsb_release -a && free -h
Linux localhost 4.15.0-176-generic #185-Ubuntu SMP Tue Mar 29 17:40:04 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.6 LTS
Release:	18.04
Codename:	bionic
              total        used        free      shared  buff/cache   available
Mem:           984M        201M        311M        6.3M        472M        635M
Swap:          511M          0B        511M
```

**Comment by githubsaturn** [comment]
If you run `serversetup` successfully, your HTTPS is activated and you cannot use `http://ip:3000` anymore, you'll have to use `captain.domain.com` to access your domain. What do you see when you run these from your local machine?

```bash
curl -v http://captain.domain.com
# and
curl -v https://captain.domain.com
```

**Comment by AaronPresley** [comment]
Thanks for your time @githubsaturn. I just spun up a new instance and was able to confirm I'm having the same issue.

HOWEVER the `curl` commands you recommended did demonstrate that HTML was being rendered on the `https` call. I found this incredibly confusing because it just loads and loads on my browser (Brave). I decided to open the URL in Safari and IT WORKS 🙌

No clue why Brave spins and spins, but it does load in an incognito tab. I'm guessing it's a browser cache issue of some kind... I'm able to debug the problem from here; it seems completely unrelated to CapRover (as expected).

Thanks for the nudge in the right direction 🙏  I'm so excited to explore CapRover more.
